cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/ChangeExecutionConcurrencyParameters.java

```diff
@@ -29,13 +30,15 @@
     validParameterNames.add(CONCURRENT_PARTITION_MOVEMENTS_PER_BROKER_PARAM);
     validParameterNames.add(CONCURRENT_INTRA_BROKER_PARTITION_MOVEMENTS_PARAM);
     validParameterNames.add(CONCURRENT_LEADER_MOVEMENTS_PARAM);
+    validParameterNames.add(MAX_PARTITION_MOVEMENTS_IN_CLUSTER_PARAM);
```

Comment:
Can we update the JavaDoc of `AdminParameters` to show the use of this new parameter?

Reply:
Done!